referencing/faq.md

```diff
@@ -0,0 +1,74 @@
+---
+layout: default
+title: Referencing FAQ
+nav_order: 1
+parent: Using References
+---
+
+# Referencing FAQ
+
+## Do I need to read the entire referencing guide?
+
+If your OpenAPI Description (OAD):
+
+1. consists only of a single JSON or YAML document
+1. contains only references that are fragments (beginning with a `"#"`, e.g. `{"$ref": "#/components/responses/oops"}`)
+1. does not have any other keywords in the same object as any `"$ref"`
+1. (OAS 3.1 only) does *not* use `"$id"`if you are using OAS 3.1, you are  *not* using the `"$id"` keyword Schema Objects
+1. (OAS 3.1 only) does *not* use `"$dynamicRef"` or `"$dynamicAnchor"`
+
+Then your use of references is simple enough that your references will do what you expect with any tool that supports them at all.  You probably do not need to read the rest of this page.
+
+## Can an OAD be *completely* "de-referenced" by "removing" all references?
+
+In general, no.  But in many common cases, yes, and tools exist that will attempt this.
+
+There are two types of reasons why de-referenceing might not be possible:
+
+* Technical reasons:
+   * Cyclic references are allowed in Schema Objects, and are used to describe recursive tree structures.  JSON Schema itself is a recursive tree structure, so the JSON Schema metaschemas cannot be fully de-referenced (see draft-07 or earlier for how this works with `"$ref"`; draft 2020-12 uses a different technique)
```

Comment:
This point about recursive references is important, and implies we *can't* ship a completely normalized document (all references resolved and inlined), and that some level of at least intra-document indirection must be supported. (eg; tools must be *slightly* more complicated than simple JSON / YAML parsers)